src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -173,6 +173,9 @@ TEST(OVClassBasicTest, smoke_createDefault) {
 
 TEST_P(OVClassBasicTestP, registerExistingPluginThrows) {
     ov::Core ie = createCoreWithTemplate();
+    try {
+        ie.register_plugin(pluginName, target_device);
+    } catch (...) {}
```

Comment:
Why required? I suppose it should throw exception  just from the first time because all known plugins already registered